packages/database/src/utilities/migration.js

```diff
@@ -210,16 +210,11 @@ export function populateEntityBounds(db) {
 
 /* eslint-disable no-unused-vars */
 
-// Get a dashboard report by id
-async function getDashboardReportById(db, id) {
-  const { rows: dashboardReports } = await db.runSql(`
-      SELECT * FROM "dashboardReport"
-      WHERE id = '${id}';
-  `);
-  return dashboardReports[0] || null;
-}
-
-// Add a dashboard report to a dashboard group
+/**
+ * @deprecated
+ * Tables "dashboardReport" and "dashboardGroup" have been dropped.
+ * Please use "dashboard", "dashboard_relation" and "dashboard_item"
+ */
 export function addReportToGroups(db, reportId, groupCodes) {
```

Comment:
I'm a bit reluctant to remove these utils because they are being used in some migrations, and I'm not sure if there's going to be any side effect if I remove migration files? (Perhaps when someone want to set up a database from a very old version of the db). So that's why I just deprecate these methods using "dashboardGroup" or "dashboardReport" here.